### PR TITLE
Initial layout

### LINKED ISSUE #3: Set up base UI layout
- [x] Create header, nav links, and homepage content.
- [x] Create < FormPage /> Component
- [x] Create < ResultPage /> Component

### PR DESCRIPTION
**Pull request by kimmccaskill**
## What functionality does this accomplish?
closes #3 

**Description:**
- Adds more of the base UI.  This includes the Result page as well as some basic code to set up state for the future

## What did you struggle on to complete?
- I struggled to get form state working completely but this was a stretch goal
- We will need to do some research on react native navigation to fully implement the nav tabs at the bttom as well as maintaining the header

### Test Coverage Percentage: x%
- [x] No Tests have been changed
- [ ] New Tests have been implemented
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):

## Checklist:
- [ ] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Helpful Resources:
* Resource link AND small description of info gathered/learned

## Please include a gif of how you feel about this branch:

alt_text

### CODE REVIEW

File: Components/Form/Form.js [hunk 1]
```diff
@@ -1,42 +1,73 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { StyleSheet, View, Modal, Text, Picker, Button } from 'react-native'
 
-const Form = ({visible}) => {
+const Form = ({visible, setShowForm, setShowResult}) => {
+  const [restaurantType, setRestaurantType] = useState('')
```
Comment (kimmccaskill) [inline Components/Form/Form.js:5]: Lines 4-6 as well as the 3 handlers below are a WIP

File: Components/Form/Form.js [hunk 2]
```diff
@@ -1,42 +1,73 @@
...
         <View style={styles.pickerContainer}>
           <Text>Travel</Text>
           <Picker
             itemStyle={{height: 44}}
+            selectedValue={travelType}
+            // onValueChange={formInputHandler}
           >
-            <Picker.Item label="Walk"/>
-            <Picker.Item label="Drive"/>
+            <Picker.Item label="Walk" value="walk"/>
```
Comment (kimmccaskill) [inline Components/Form/Form.js:48]: Value of these two options will change

File: Components/ResultPage/ResultPage.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          <Text>303-420-6969</Text>
+          <Text>69639 Burrito Way - Open in Maps</Text>
+        </View>
+        <ScrollView 
+          showsHorizontalScrollIndicator={false} 
+          horizontal={true} 
+          style={{width: '100%'}}
+        >
+          <View style={styles.imgContainer}>
+            <Image
```
Comment (kimmccaskill) [inline Components/ResultPage/ResultPage.js:23]: These are placeholder images that will change after hitting back end for images

File: Components/Form/Form.js [hunk 4]
```diff
@@ -1,42 +1,73 @@
...
+import React, { useState } from 'react'
 import { StyleSheet, View, Modal, Text, Picker, Button } from 'react-native'
 
-const Form = ({visible}) => {
+const Form = ({visible, setShowForm, setShowResult}) => {
+  const [restaurantType, setRestaurantType] = useState('')
+  const [travelType, setTravelType] = useState('')
+  const [cost, setCost] = useState('')
+
+  const restaurantHandler = (type) => {
```
Comment (CHaiz15) [inline Components/Form/Form.js:9]: I think we definitely can clean up these handlers to be a little more DRY 👍

**Pull request merged**